### ISSUE TITLE
FabricBakedModel's have an EMPTY BlockView

### ISSUE THREAD

**Issue by RuffleSteels** [comp500/Indium#322]
### Confirmation

- [X] I have tested the bug without Sodium and Indium.
- [X] I have kept all my mods up to date.

### Bug Description

I have a FabricBakedModel which has emitBlockQuads() and I use the blockView param to change the model dependant on the biome, however when sodium and or indium is installed the blockView param seems to be always EMPTY, thus it crashes and errors as I am trying to use it. I would have thought Indium would have implemented support for FabricBakedModels as they are in the fabapi rendering, however this seems not the case.

### Mod Version

Indium-1.0.31

### Minecraft Version

1.20.4

### Attach Crash Report

Time: 2024-07-29 12:48:08
Description: Encountered exception while building chunk meshes

java.lang.NullPointerException: Cannot invoke "net.minecraft.registry.entry.RegistryEntry.getKey()" because the return value of "net.minecraft.world.BlockView.getBiomeFabric(net.minecraft.util.math.BlockPos)" is null
	at com.oscimate.oscimate_soulflame.test.TestModel$1.getQuads(TestModel.java:79)
	at com.oscimate.oscimate_soulflame.test.TestModel.getQuads(TestModel.java:154)
	at net.minecraft.client.render.model.WeightedBakedModel.getQuads(WeightedBakedModel.java:535)
	at net.minecraft.client.render.model.MultipartBakedModel.getQuads(MultipartBakedModel.java:570)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.pipeline.BlockRenderer.getGeometry(BlockRenderer.java:90)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.pipeline.BlockRenderer.renderModel(BlockRenderer.java:72)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.redirect$bem000$indium$onRenderBlock(ChunkBuilderMeshingTask.java:1045)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:103)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:39)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkJobTyped.execute(ChunkJobTyped.java:44)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder$WorkerRunnable.run(ChunkBuilder.java:186)
	at java.base/java.lang.Thread.run(Thread.java:833)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.oscimate.oscimate_soulflame.test.TestModel$1.getQuads(TestModel.java:79)
	at com.oscimate.oscimate_soulflame.test.TestModel.getQuads(TestModel.java:154)
	at net.minecraft.client.render.model.WeightedBakedModel.getQuads(WeightedBakedModel.java:535)
	at net.minecraft.client.render.model.MultipartBakedModel.getQuads(MultipartBakedModel.java:570)

-- Block being rendered --
Details:
	Block: Block{minecraft:fire}[age=15,east=false,north=false,south=false,up=false,west=false]
	Block location: World: (-99,56,321), Section: (at 13,8,1 in -7,3,20; chunk contains blocks -112,0,320 to -97,255,335), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Chunk section: RenderSection at chunk (-7, 3, 20) from (-112, 48, 320) to (-97, 63, 335)
	Render context volume: BlockBox{minX=-114, minY=46, minZ=318, maxX=-95, maxY=65, maxZ=337}
Stacktrace:
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:103)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:39)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkJobTyped.execute(ChunkJobTyped.java:44)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder$WorkerRunnable.run(ChunkBuilder.java:186)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- Uptime --
Details:
	JVM uptime: 55.015s
	Wall uptime: 27.671s
	High-res time: 23.970s
	Client ticks: 169 ticks / 8.450s
Stacktrace:
	at net.minecraft.client.MinecraftClient.addDetailsToCrashReport(MinecraftClient.java:2544)
	at net.minecraft.client.MinecraftClient.printCrashReport(MinecraftClient.java:1004)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:911)
	at net.minecraft.client.main.Main.main(Main.java:243)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)

**Comment by PepperCode1** [comment]
Indium does not pass an empty level to the model (specifically `emitBlockQuads`). It passes Sodium's `WorldSlice` directly. If it passed an empty level, a lot of mods wouldn't work, but they do. `getBiomeFabric` is implemented by Sodium's `WorldSlice` too, so if you are having issues with it, report that to Sodium. However, the fact that you are somehow accessing the level from within `getQuads`, which doesn't receive a level, is suspicious and is likely the cause of your issue.

**Comment by RuffleSteels** [comment]
That seemed to be sorted by making the `isVanillaAdaper()` false, but however, in the most recent fabric api version for 1.21, they seem to have changed something else which causes a crash when using that exact thing (a false `isVanillaAdapter()`), while keeping the same most recent indium and sodium 1.21 versions.

```
`---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 2024-08-12 22:59:59
Description: Tessellating block in world - Indium Renderer

java.lang.NoSuchMethodError: 'void net.fabricmc.fabric.impl.renderer.VanillaModelEncoder.emitBlockQuads(net.minecraft.client.render.model.BakedModel, net.minecraft.block.BlockState, java.util.function.Supplier, net.fabricmc.fabric.api.renderer.v1.render.RenderContext, net.fabricmc.fabric.api.renderer.v1.mesh.QuadEmitter)'
	at net.minecraft.client.render.model.BakedModel.emitBlockQuads(BakedModel.java:1039)
	at com.oscimate.firorize.test.TestModel.emitBlockQuads(TestModel.java:238)
	at net.minecraft.client.render.model.WeightedBakedModel.emitBlockQuads(WeightedBakedModel.java:574)
	at net.minecraft.client.render.model.MultipartBakedModel.emitBlockQuads(MultipartBakedModel.java:601)
	at link.infra.indium.renderer.render.TerrainRenderContext.tessellateBlock(TerrainRenderContext.java:131)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.redirect$zep000$indium$onRenderBlock(ChunkBuilderMeshingTask.java:543)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:103)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:39)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkJobTyped.execute(ChunkJobTyped.java:44)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder$WorkerRunnable.run(ChunkBuilder.java:186)
	at java.base/java.lang.Thread.run(Thread.java:1570)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.render.model.BakedModel.emitBlockQuads(BakedModel.java:1039)
	at com.oscimate.firorize.test.TestModel.emitBlockQuads(TestModel.java:238)
	at net.minecraft.client.render.model.WeightedBakedModel.emitBlockQuads(WeightedBakedModel.java:574)

-- Block being tessellated --
Details:
	Block: Block{minecraft:fire}[age=0,east=false,north=false,south=false,up=false,west=false]
	Block location: World: (-125,81,466), Section: (at 3,1,2 in -8,5,29; chunk contains blocks -128,0,464 to -113,255,479), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
Stacktrace:
	at link.infra.indium.renderer.render.TerrainRenderContext.tessellateBlock(TerrainRenderContext.java:131)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.redirect$zep000$indium$onRenderBlock(ChunkBuilderMeshingTask.java:543)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:103)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:39)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkJobTyped.execute(ChunkJobTyped.java:44)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder$WorkerRunnable.run(ChunkBuilder.java:186)
	at java.base/java.lang.Thread.run(Thread.java:1570)

-- Block being rendered --
Details:
	Block: Block{minecraft:fire}[age=0,east=false,north=false,south=false,up=false,west=false]
	Block location: World: (-125,81,466), Section: (at 3,1,2 in -8,5,29; chunk contains blocks -128,0,464 to -113,255,479), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Chunk section: RenderSection at chunk (-8, 5, 29) from (-128, 80, 464) to (-113, 95, 479)
	Render context volume: BlockBox{minX=-130, minY=78, minZ=462, maxX=-111, maxY=97, maxZ=481}
Stacktrace:
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:103)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.tasks.ChunkBuilderMeshingTask.execute(ChunkBuilderMeshingTask.java:39)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkJobTyped.execute(ChunkJobTyped.java:44)
	at me.jellysquid.mods.sodium.client.render.chunk.compile.executor.ChunkBuilder$WorkerRunnable.run(ChunkBuilder.java:186)
	at java.base/java.lang.Thread.run(Thread.java:1570)

-- Uptime --
Details:
	JVM uptime: 56.682s
	Wall uptime: 31.505s
	High-res time: 28.825s
	Client ticks: 361 ticks / 18.050s
Stacktrace:
	at net.minecraft.client.MinecraftClient.addDetailsToCrashReport(MinecraftClient.java:2501)
	at net.minecraft.client.MinecraftClient.printCrashReport(MinecraftClient.java:949)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:902)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['Player98'/59, l='ClientLevel', x=-118.13, y=81.88, z=470.65]]
	Chunk stats: 1024, 196
	Level dimension: minecraft:the_nether
	Level spawn location: World: (498,-46,2199), Section: (at 2,2,7 in 31,-3,137; chunk contains blocks 496,0,2192 to 511,255,2207), Region: (0,4; contains chunks 0,128 to 31,159, blocks 0,0,2048 to 511,255,2559)
	Level time: 208931 game time, 123869 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
	Tracked entity count: 132

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-gametest-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, firorize, indium, modmenu, sodium

-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 22.0.2, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 472899256 bytes (450 MiB) / 952107008 bytes (908 MiB) up to 8556380160 bytes (8160 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 10
	Microarchitecture: Coffee Lake
	Frequency (GHz): 3.19
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1060 6GB
	Graphics card #0 vendor: NVIDIA
	Graphics card #0 VRAM (MiB): 6144.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.15.5241
	Memory slot #0 capacity (MiB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 36027.93
	Virtual memory used (MiB): 24118.18
	Swap memory total (MiB): 3392.96
	Swap memory used (MiB): 120.07
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 1083825.13, total: 1906471.00
	Space in storage for workdir (MiB): available: 1083825.13, total: 1906471.00
	JVM Flags: 0 total; 
	Fabric Mods: 
		fabric-api: Fabric API 0.102.0+1.21
		fabric-api-base: Fabric API Base 0.4.42+6573ed8cd1
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.68+b5597344d1
		fabric-biome-api-v1: Fabric Biome API (v1) 13.0.29+5bd9f1bcd1
		fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a7d1
		fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8cd1
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.52+0af3f5a7d1
		fabric-client-tags-api-v1: Fabric Client Tags 1.1.15+6573ed8cd1
		fabric-command-api-v1: Fabric Command API (v1) 1.2.49+f71b366fd1
		fabric-command-api-v2: Fabric Command API (v2) 2.2.28+6ced4dd9d1
		fabric-commands-v0: Fabric Commands (v0) 0.2.66+df3654b3d1
		fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.16+b5597344d1
		fabric-convention-tags-v1: Fabric Convention Tags 2.0.19+7f945d5bd1
		fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.5.0+c5e2b5c6d1
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a7d1
		fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.27+6a6dfa19d1
		fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.16+16c4ae25d1
		fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b99d1
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.12+6fc22b99d1
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.12+ba9dae06d1
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.53+6ced4dd9d1
		fabric-gametest-api-v1: Fabric Game Test API (v1) 2.0.4+6fc22b99d1
		fabric-item-api-v1: Fabric Item API (v1) 11.0.0+afdfc921d1
		fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.4+78017270d1
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.47+0af3f5a7d1
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.45+df3654b3d1
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.12+6c1df360d1
		fabric-loot-api-v2: Fabric Loot API (v2) 3.0.14+3f89f5a5d1
		fabric-loot-api-v3: Fabric Loot API (v3) 1.0.2+3f89f5a5d1
		fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8cd1
		fabric-model-loading-api-v1: Fabric Model Loading API (v1) 2.0.0+fe474d6bd1
		fabric-networking-api-v1: Fabric Networking API (v1) 4.2.2+60c3209bd1
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.2.0+a551f7a4d1
		fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8cd1
		fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.12+65089712d1
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.1.2+60c3209bd1
		fabric-renderer-api-v1: Fabric Renderer API (v1) 3.4.0+c705a49cd1
		fabric-renderer-indigo: Fabric Renderer - Indigo 1.7.0+c705a49cd1
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.68+df3654b3d1
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2ed1
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+b5597344d1
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.71+df3654b3d1
		fabric-rendering-v1: Fabric Rendering (v1) 5.0.5+df16efd0d1
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.3.0+8dc279b1d1
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.3.0+56599129d1
		fabric-screen-api-v1: Fabric Screen API (v1) 2.0.24+b5597344d1
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.86+b5597344d1
		fabric-sound-api-v1: Fabric Sound API (v1) 1.0.23+6573ed8cd1
		fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.16+3dccd343d1
		fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.0+176f9036d1
		fabricloader: Fabric Loader 0.15.11
		firorize: Firorize 2.0
		indium: Indium 1.0.34+mc1.21
		java: OpenJDK 64-Bit Server VM 22
		minecraft: Minecraft 1.21
		mixinextras: MixinExtras 0.4.1
		modmenu: Mod Menu 11.0.1
		sodium: Sodium 0.5.11+mc1.21
	Launched Version: Fabric
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: NVIDIA GeForce GTX 1060 6GB/PCIe/SSE2 GL version 3.2.0 NVIDIA 552.41, NVIDIA Corporation
	Window size: 854x480
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Render Distance: 12/12 chunks
	Resource Packs: vanilla, fabric, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-gametest-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, firorize, indium, modmenu, sodium
	Current Language: en_us
	Locale: en_GB
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 12x Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
	Server Running: true
	Player Count: 1 / 8; [ServerPlayerEntity['Player98'/59, l='ServerLevel[New World]', x=-118.13, y=81.88, z=470.65]]
	Active Data Packs: vanilla, fabric, fabric-convention-tags-v2, fabric-gametest-api-v1
	Available Data Packs: bundle, fabric, fabric-convention-tags-v2, fabric-gametest-api-v1, trade_rebalance, vanilla
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: 3090187447101834687`
```


I suspect you may need to tweak some code to fit with the new fabapi version, I have looked at the changes but couldn't make out enough to create a pull request to try solve this issue.

**Comment by RuffleSteels** [comment]
ah i see this is already being worked on in #324 i will close this thread
